packages/rs-dpp/src/data_contract/document_type/document_type.rs

```diff
@@ -36,10 +37,20 @@ pub struct DocumentType {
     pub indices: Vec<Index>,
     #[serde(skip)]
     pub index_structure: IndexLevel,
+    /// Flattened properties flatten all objects for quick lookups for indexes
```

Comment:
How about naming it DocumentDefinition instead? When I hear "type", I usually think about the name, like "a document of type Contact"

Reply:
made a task